src/mapper/value-mapper.factory.ts

```diff
@@ -26,6 +26,13 @@ export const valueMapperFactory = async () => {
     config.enableInternalEmailPlusAddressing,
     config.enableExternalEmailPlusAddressing
   ])
+  const [
```

Comment:
For some reason, it's not possible to put more than 10 promises in the array, so I had to split them. Do you know why/how to fix it? @janjilek